dask_ml/metrics/regression.py

```diff
@@ -24,8 +30,12 @@ def _check_reg_targets(y_true, y_pred, multioutput):
 
 @derived_from(sklearn.metrics)
 def mean_squared_error(
-    y_true, y_pred, sample_weight=None, multioutput="uniform_average", compute=True
-):
+    y_true: ArrayLike,
+    y_pred: ArrayLike,
+    sample_weight: Optional[ArrayLike] = None,
+    multioutput: Optional[str] = "uniform_average",
```

Comment:
Would be good to check with scikit-learn what the actual supported types are. Can this be an array, which just isn't implemented yet?

Reply:
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_squared_error.html

`multi_output` is a string